subsys/net/lib/lwm2m/lwm2m_util.h

```diff
@@ -22,4 +22,15 @@ int lwm2m_atof(const char *input, double *out);
 /* convert float to string */
 int lwm2m_ftoa(double *input, char *out, size_t outlen, int8_t dec_limit);
 
+uint16_t lwm2m_atou16(const uint8_t *buf, uint16_t buflen, uint16_t *len);
+
+/* Forms a string from the path given as a structure
+ * Level zero seems to mean NONE, not the root path.
+ *
+ * returns used buffer space with '\0'
+ */
+int lwm2m_path_to_string(char *buff, size_t buff_size, struct lwm2m_obj_path *input, int level_max);
```

Comment:
`buf`, `buf_size` would be more consistent with other functions.

Reply:
good point